features/patroni_api.feature

```diff
@@ -54,10 +54,10 @@ Scenario: check API requests for the primary-replica pair
 	And I receive a response role replica
 	When I run patronictl.py reinit batman postgres1 --force
 	Then I receive a response returncode 0
-	And I receive a response output "Succesful reinitialize on member postgres1"
+	And I receive a response output "Success: reinitialize for member postgres1"
 	When I run patronictl.py restart batman postgres0 --force
 	Then I receive a response returncode 0
-	And I receive a response output "Succesful restart on member postgres0"
+	And I receive a response output "Successful restart on member postgres0"
```

Comment:
We should also change the restart operation to produce Success: and Failed: messages, the same way as reinitialize.